axum/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Unreleased
 
+- **added:** Add `NoContent` as a self-described shortcut for `StatusCode::NO_CONTENT`.
```

Comment:
Can you add a reference to this PR here (like below) and remove the trailing `.`?

Reply:
Fixed.